patches/chrome-android-BUILD.gn.patch

```diff
@@ -78,7 +78,7 @@ index 929b1f5742d52294fb0dfa03601662744911d036..b6dbf051424fae3541b5a17b977d293f
      "//components/payments/content/android:service_java",
      "//third_party/androidx:androidx_browser_browser_java",
    ]
-+  sources += brave_java_base_module_sources deps += brave_java_base_module_deps
++  sources += brave_java_base_module_sources deps += brave_java_base_module_deps sources -= brave_java_base_module_sources_exclude deps -= ["$google_play_services_package:google_firebase_firebase_iid_java", "$google_play_services_package:google_firebase_firebase_messaging_java",]
```

Comment:
I had to do `deps -= ["$google_play_services_package:google_firebase_firebase_iid_java", "$google_play_services_package:google_firebase_firebase_messaging_java",]`, instead of doing both excludes as `sources -=` because `$google_play_services_package` is defined inside `build/config/android/config.gni` and it errors with a deps loop if I try to include `build/config/android/config.gni` inside brave's file.